can_explorer_client/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```

Comment:
I think Raiyan said he was forwarding port 8000 from the host to the VM, not 3000

Reply:
If we want to run a Django + React app, it looks like we will need two ports anyway (one running the backend and another running the React front-end). This [guide](https://www.digitalocean.com/community/tutorials/build-a-to-do-application-using-django-and-react) goes into the idea in more detail.

Reply:
Sure, Raiyan could you add this to the Vagrantfile then? I'm fine with exposing both port 8000 and 3000.